Makefile

```diff
@@ -0,0 +1,35 @@
+REPO               = knative-lambda-runtime
+REPO_DESC          = TriggerMesh Knative Lambda Runtime
+RUNTIMES           = java8 node-10.x node-4.x python-2.7 python-3.7 ruby-2.5
+
+BASE_DIR          ?= $(CURDIR)
+
+DOCKER            ?= docker
+IMAGE_REPO        ?= gcr.io/triggermesh
+IMAGE_TAG         ?= latest
+IMAGE_SHA         ?= $(shell git rev-parse HEAD)
+
+.PHONY: help images cloudbuild-test cloudbuild
+
+all: images
+
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*?## "; printf "\n$(REPO_DESC)\nUsage:\n  make \033[36m<source>\033[0m\n"} /^[a-zA-Z0-9._-]+:.*?## / {printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+IMAGES = $(foreach r,$(RUNTIMES),$(r).image)
+images: $(IMAGES) ## Build the Docker images
+$(IMAGES): %.image:
+	@docker build -t $(IMAGE_REPO)/knative-lambda-$$(echo "$*" | sed -n -e "s/\([[:alnum:]]*\)\(-\)*\([0-9]*\)\(\.\)*\([0-9]*\)\(\.\)*\([0-9]*\).*/\1\3\5\7/p") $*
```

Comment:
😨

Reply:
yea. it's a bit of a stretch. 
We could get rid of the voodoo stuff if we just renamed the `folders` to their image names, f.e. `python-2.7` -> `python27`

Reply:
renaming folders would require updates in several documents that refer to KLR runtime yaml files as well as in CLI bits that are also using KLR yamls. Maybe we could just list folders by their current names to avoid renaming?

Reply:
> Maybe we could just list folders

do you mean link?

Reply:
nope, I need to think about what could be the solution, please don't merge yet. Although, maybe having the unified folder name structure actually worth updating all related docs and CLI.

Reply:
@tzununbekov i'll revert my folder renaming change.

Reply:
@tzununbekov i've reverted the folder rename change. please take a look

Reply:
well, the only possible way other than moving current paths is walking through the existing folders and grep-ing runtime.yaml for `FROM gcr.io/triggermesh` string to see which image is being used by the runtime. This also feels really hackish, so if we decide to go with the renaming approach, I will not argue - we'll just need to update a couple of files in the tm CLI and a bunch of them in the current repository.

Reply:
> feels really hackish

That's true.

> need to update a couple of files in the tm CLI and a bunch of them in the current repository.

I can work on this change. Are there any docs that would need updating?

Reply:
I think it would be easier for me to update those references. Give me 20 minutes, I'll link related PRs